assets/js/login.js

```diff
@@ -0,0 +1,19 @@
+require('../scss/login.scss')
+
+$('#login-form .form-control').on('blur input focus', function () {
+  var $field = $(this).closest('.form-group')
+  if ($(this).val()) {
+    $field.addClass('filled')
+  } else {
+    $field.removeClass('filled')
+  }
+})
+
+$('#login-form .form-control').on('focus', function () {
+  var $field = $(this).closest('.form-group')
+  if ($(this)) {
+    $field.addClass('filled')
+  } else {
+    $field.removeClass('filled')
+  }
+})
```

Comment:
An event handler for the `focus` event (on the same elements) is set in both lines 3 and 12. Unless I'm missing something the second event handler is superfluous.

Reply:
New commit will address this.